scripts/functions/requirements/osx_brew

```diff
@@ -290,6 +311,8 @@ requirements_osx_brew_libs_default()
 
   requirements_osx_brew_check_custom homebrew/dupes # for: zlib
 
+  undesired_check openssl098
```

Comment:
not sure it should be in `custom` or rather in `before` or `define`? or maybe we should add `before_install`? or even a new step `uninstall`? - if `custom` is called always - just leave it :)